src/wast-lexer.cc

```diff
@@ -96,10 +96,8 @@ namespace wabt {
 
 struct WastLexer::LexToken {
   Location loc_;
```

Comment:
Actually, this needs to be initialized too -- the current default constructor does not initialize the values. But that can be fixed in another PR.